tests_integration/nativeapp/test_deploy.py

```diff
@@ -102,9 +102,13 @@ def test_nativeapp_deploy(
     "command,contains,not_contains",
     [
         # deploy --prune removes remote-only files
-        ["app deploy --prune", ["stage/manifest.yml"], ["stage/README.md"]],
+        [
+            "app deploy --prune --no-validate",
+            ["stage/manifest.yml"],
+            ["stage/README.md"],
+        ],
         # deploy removes remote-only files (--prune is the default value)
-        ["app deploy", ["stage/manifest.yml"], ["stage/README.md"]],
+        ["app deploy --no-validate", ["stage/manifest.yml"], ["stage/README.md"]],
```

Comment:
Did you have to add this everywhere to avoid touching all the snapshots?

Reply:
no, it's because these examples aren't valid apps (some don't upload the manifest, some don't upload the setup script, etc). Those operations are valid by themselves, but since the files on the stage at that point don't make a valid app, the validation function fails. Since it's not the goal of the test, I omitted validation here to restore the original behaviour

Reply:
That's fair, resolving.